plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -61,6 +61,7 @@
     private static final String PARQUET_MAX_READ_BLOCK_SIZE = "parquet_max_read_block_size";
     private static final String PARQUET_WRITER_BLOCK_SIZE = "parquet_writer_block_size";
     private static final String PARQUET_WRITER_PAGE_SIZE = "parquet_writer_page_size";
+    private static final String STATISTICS_ENABLED = "statistics_enabled";
```

Comment:
This is consistent with hive's `statistics_enabled`, but indeed not consistent with property name.

i would go with current naming as proposed in this PR.